cpp/tests/ast/transform_tests.cpp

```diff
@@ -343,6 +343,74 @@ TEST_F(TransformTest, ImbalancedTreeArithmeticDeep)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, result->view(), verbosity);
 }
 
+TEST_F(TransformTest, DeeplyNestedArithmeticLogicalExpression)
+{
+  // Test logic for deeply nested arithmetic and logical expressions.
+  constexpr int64_t kLeftDepthLevel  = 100;
+  constexpr int64_t kRightDepthLevel = 75;
```

Comment:
We generally use `snake_case` for variables.

Reply:
changed as suggested